src/components/user-profile.component.js

```diff
@@ -124,6 +125,7 @@ export const UserProfile = ({
       </Text>
     </View>
     <View style={styles.details}>
+      {user.hasOwnProperty('public_repos') &&
```

Comment:
Can we just do `user.public_repos` here?

Reply:
There may be zero repositories, and then nothing will be shown, right?

Reply:
Yeah that's true 🤔 

Would again prefer using `_.has()` from lodash but let's discuss that later if we decide to include the whole library